invenio_rdm_records/ext.py

```diff
@@ -24,6 +25,13 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
+        self.metadata_extensions = MetadataExtensions(
+            app.config['RDM_RECORDS_METADATA_EXTENSIONS']
+        )
+        before_record_index.connect(
```

Comment:
The receiver will receive all calls for indexing, so right now it will work but as soon as we have other records type this function will be called for them as well. The ``dynamic_conncet()`` will connect based on the ``$schema``

Reply:
Ah, very nice!